expresservapp/expressapp.js

```diff
@@ -407,6 +414,20 @@ apiRouter.route("/admin/delFeedback")
         });
     });
 
+apiRouter.route("/picsFromInstagram")
+    .get(function(req, res) {
+        var instaPicsUrls = [];
+        requestify.get('https://api.instagram.com/v1/users/self/media/recent/?access_token=10020038878.91ffc57.4285141862d14b4880a447fbd04ecc1a')
```

Comment:
Please extract API url and access token into separate global variables (these 2 are subjects to change, and it's always easier to change such values in one place instead of seraching through code)